integration/client_test.go

```diff
@@ -202,27 +209,35 @@ func TestUpdateValidation(t *testing.T) {
 			t.Fatalf("Update(%v): %v != %v, want %v", tc.userID, err, want, tc.want)
 		}
 		if tc.want {
-			if err := env.Queue.AdvanceEpoch(); err != nil {
-				t.Errorf("AdvanceEpoch: %v", err)
+			txn, err := env.Factory.NewDBTxn(bctx)
```

Comment:
use the function I referenced pulling out above.  This seems like a lot of boiler plate code